content/200-concepts/200-database-connectors/07-mongodb.mdx

```diff
@@ -3,17 +3,184 @@ title: 'MongoDB'
+
+```
+mongodb://USERNAME:PASSWORD@HOST/DATABASE
+```
+
+The following components make up the _base URL_ of your database:
+
+| Name     | Placeholder | Description                                                                                                                                                                                                                                                                                                                                                |
+| :------- | :---------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| User     | `USERNAME`  | Name of your database user, e.g. `janedoe`                                                                                                                                                                                                                                                                                                                 |
+| Password | `PASSWORD`  | Password for your database user                                                                                                                                                                                                                                                                                                                            |
+| Host     | `HOST`      | The host where a [`mongod`](https://docs.mongodb.com/manual/reference/program/mongod/#mongodb-binary-bin.mongod) instance is running. If you are runnnig a sharded cluster this will a [`mongos`](https://docs.mongodb.com/manual/reference/program/mongos/#mongodb-binary-bin.mongos) instance. This can be a hostname, IP address or UNIX domain socket. |
+| Port     | `PORT`      | Port on which your database server is running, e.g. `1234`. If none is provided the default `27017` is used.                                                                                                                                                                                                                                               |
+| Database | `DATABASE`  | Name of the database to use. If none is specified but the `authSource` option is set then the `authSource` database name is used. If neither the database in the connection string nor the `authSource` option is specified then it defautls to `admin`                                                                                                    |
+
+<Admonition type="info">
+
+You must [percentage-encode special characters](../../../reference/database-reference/connection-urls#special-characters).
```

Comment:
Left this in here as I'm not sure - it _looks_ like yes, you need to percentage-encode: https://stackoverflow.com/questions/7486623/mongodb-password-with-in-it